test/test_operators.py

```diff
@@ -1,5 +1,5 @@
 import unittest
-from test.util import TVMTest
```

Comment:
what's going on here? I think this will break CI

Reply:
Strange. Will fix this.